dgl_ptm/.githooks/pre-commit

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+echo "Script $0 triggered ..."
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+echo "Starting prospector analysis using configuration from .prospector.yml..."
+
+# quietly run prospector
+prospector 1>/dev/null
+
+# use return code to abort commit if necessary
+if [ $? != "0" ]; then
+    echo "Commit aborted. Run 'prospector' to see the errors."
+    exit 1
+fi
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+echo "Starting isort analysis using configuration from setup.cfg..."
+
+# recursively run isort on dgl_ptm/ directory, don't try to automatically fix anything
+isort --recursive --check-only dgl_ptm
+
+if [ $? != "0" ]; then
+    echo "Commit aborted."
+    echo "  Run 'isort --check-only --diff dgl_ptm' to see what's wrong."
```

Comment:
Shouldn't this (and the next line) be recursive as well?

Reply:
Apparently, recursive is no longer required as per https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html#-recursive-or-rc:

--recursive or -rc
Prior to version 5.0.0, isort wouldn't automatically traverse directories. The --recursive option was necessary to tell it to do so. In 5.0.0 directories are automatically traversed for all Python files, and as such this option is no longer necessary and should simply be removed.